File: 백준/Bronze/2576. 홀수/홀수.java
import java.util.*;
import java.io.*;

public class Main {
	static int sum;
	static int min = Integer.MAX_VALUE;

	public static void main(String[] args) throws IOException {
		BufferedReader br = new BufferedReader(new InputStreamReader(System.in));
		boolean AllOdd = true;
		for (int i = 0; i < 7; i++) {
			int k = Integer.parseInt(br.readLine());
			if (k % 2 != 0) {
				sum += k;
				if (min > k) {
					min = k;
				}
				AllOdd = false;
			}
		}
		if (!AllOdd) {
			System.out.println(sum);
			System.out.println(min);
		} else {
			System.out.println("-1");
		}
	}
}

File: 백준/Bronze/2576. 홀수/홀수.py
mx = 200
sum = 0
for _ in range(0,7):
    a = int(input())
    if a % 2 != 0:
        sum += a
        mx = min(mx, a)
if sum != 0:
    print(sum)
    print(mx)
else:
    print(-1)